config/jobs/kubernetes/release/release-config.yaml

```diff
@@ -304,6 +304,42 @@ presubmits:
       testgrid-num-failures-to-alert: '10'
       testgrid-alert-email: release-managers+alerts@kubernetes.io
       testgrid-num-columns-recent: '30'
+  - name: pull-release-image-k8s-ci-builder
```

Comment:
It looks like linked PR is for `k8s-cloud-builder`, but this is for `k8s-ci-builder`

Reply:
true! oh boy names confusion! good catch, will update

Reply:
fixed